### PR TITLE
Feature 2147 workflow doc

### LINKED ISSUE #2147: Internal: Enhance GitHub Workflow documentation
## Describe the Task ##
Enhance the  GitHub Workflow documentation by adding information about how the Cycle selection can only be added after the Issue is created and how  the Cycle and Issue selection can only be added after the PR is created.  

Use the following for PRs from George from Slack in the "[Open a pull request](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html#open-a-pull-request)" section:
1. Before the pull request is created, one can select the development project, but cannot select a cycle: <add image>
2. After the PR is created, more options appear until the Project section: <add image>
3. Click the status pull down and select "Review": <add image>
4. Click on "+1 more" then click "Choose an iteration" under "Cycle" and select the current development cycle: <add image>
5. Click on the gear next to "Development," type the issue number that corresponds to the work, then select the issue that is displayed: <add image>

For Issues (potentially modify the “[Find the GitHub issue](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html#find-the-github-issue)” section to be “Find the or Create a GitHub issue” section):
1. Before the issue is created, one can select the development project, but cannot select a cycle: <add image>
2. After the PR is created, more options appear until the Project section: <add image>
3. Click the status pull down and select "Review": <add image>
4. Click on "+1 more" then click "Choose an iteration" under "Cycle" and select the current development cycle: <add image>

### Time Estimate ###
*Estimate the amount of work required here.*
<1 day

### Sub-Issues ###
Consider breaking the task down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
None

### Funding Source ###
Request is in for Tara to assign; Waiting to hear back.

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED CYCLE ASSIGNMENT** label
- [x] Select **Milestone** as the next official version or **Future Versions**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdataio](https://github.com/dtcenter/METdataio/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Task Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements) table.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issues
Select: **Repository** level development cycle **Project** for the next official release
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by jprestop**
## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>
I reviewed the changes in ReadTheDocs:
https://metplus.readthedocs.io/en/feature_2147_workflow_doc/Contributors_Guide/github_workflow.html#find-or-create-a-github-issue
https://metplus.readthedocs.io/en/feature_2147_workflow_doc/Contributors_Guide/github_workflow.html#wo-find-issue

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
Review the changes and check for typos.

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**

- [x] Do these changes include sufficient testing updates? **[N/A]**

- [x] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Please complete this pull request review by **[20230524]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [N/A] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements) table.
- [x] Review the source issue metadata (required labels, projects, and milestone).
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization** level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [x] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: docs/Contributors_Guide/github_workflow.rst [hunk 1]
```diff
@@ -336,6 +336,46 @@ Find the GitHub issue
...
+  and review the categories of issues (e.g. Bug report, enhancement request,
+  New feature request, New use case, Sub-Issue, Task).  Find an appropriate
+  categories and click on "Get Started" next to the category.
+
+  Create a short, but descriptive title. In the 'write' window, follow the
+  directions and fill in the template.  Add any additional comments/details.
+  When filling in the template, be sure to "Define the PR metadata, as
+  permissions allow. Select: **Assignee(s), Project(s)**, and **Milestone**". 
+
+  Before an issue is created, a "Project" can be created, but there is no
```
Comment (georgemccabe) [inline docs/Contributors_Guide/github_workflow.rst]:
```suggestion
  Before an issue is created, a "Project" can be selected, but there is no
```

File: docs/Contributors_Guide/github_workflow.rst [hunk 2]
```diff
@@ -336,6 +336,46 @@ Find the GitHub issue
...
+
+  .. image:: figure/1Issue-before-created.png
+    :width: 400	     
+
+  After the issue is created, more options appear under the "Project" section.
+
+  .. image:: figure/2Issue-after-created.png
+    :width: 400
+	    
+  Click the "Status" drop down and select "Review".
```
Comment (georgemccabe) [inline docs/Contributors_Guide/github_workflow.rst]:
```suggestion
  Click the "Status" drop down and select "Todo".
```

File: docs/Contributors_Guide/github_workflow.rst [hunk 3]
```diff
@@ -672,6 +712,29 @@ Open a pull request
     * When everything looks satisfactory, click on the green 'Create pull
       request' button.
 
+    * Before a pull request is created, a "Project" can be created, but there
```
Comment (georgemccabe) [inline docs/Contributors_Guide/github_workflow.rst]:
```suggestion
    * Before a pull request is created, a "Project" can be selected, but there
```

**Review by georgemccabe — CHANGES_REQUESTED** [review]
Looks good except for a few minor edits.

File: docs/Contributors_Guide/github_workflow.rst (hunk 2 above)
Comment (jprestop) [reply docs/Contributors_Guide/github_workflow.rst]: Great catch!  Thank you so much!

**Comment by jprestop** [comment]
@georgemccabe Thanks for catching those things and for your suggestions!  I accepted your suggestions.

**Review by georgemccabe — APPROVED** [review]

**Pull request merged**